frontend/packages/integration-tests-cypress/support/a11y.ts

```diff
@@ -34,19 +34,18 @@ Cypress.Commands.add('logA11yViolations', (violations: Result[], target: string)
   cy.task('logTable', violationData);
 });
 
-Cypress.Commands.add('testA11y', (target: string) => {
+Cypress.Commands.add('testA11y', (target: string, selector?: string) => {
   cy.injectAxe();
   cy.configureAxe({
     rules: [
       { id: 'color-contrast', enabled: false }, // seem to be somewhat inaccurate and has difficulty always picking up the correct colors, tons of open issues for it on axe-core
       { id: 'focusable-content', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
       { id: 'scrollable-region-focusable', enabled: false }, // recently updated and need to give the PF team time to fix issues before enabling
-      { id: 'aria-hidden-focus', enabled: false }, // disabling until we implement correct handling of Modals, see https://dequeuniversity.com/rules/axe/3.4/aria-hidden-focus?application=axeAPI
     ],
   });
   a11yTestResults.numberChecks += 1;
   cy.checkA11y(
-    null,
+    selector || null,
```

Comment:
I don't think you need `|| null` here? Does it cause a problem leaving that off?

Reply:
fixed, removed `|| null`